saml-core-api/src/main/java/org/keycloak/saml/common/constants/JBossSAMLConstants.java

```diff
@@ -55,6 +55,7 @@
     RESPONSE__PROTOCOL(PROTOCOL_NSURI, "Response"),
     SCOPING(PROTOCOL_NSURI, "Scoping"),
     SESSION_INDEX(PROTOCOL_NSURI, "SessionIndex"),
+    SESSION_NOT_ON_OR_AFTER(PROTOCOL_NSURI, "SessionNotOnOrAfter"),
```

Comment:
Please move this constant to `org.keycloak.saml.processing.core.parsers.saml.assertion.SAMLAssertionQNames` as an `ATTR_SESSION_NOT_ON_OR_AFTER` constant similarly to other `ATTR_*` constants declared there. `JBossSAMLConstants` is eventually going to be removed.